pkg/compactor/bucket_compactor.go

```diff
@@ -248,13 +248,23 @@ func (c *BucketCompactor) runCompactionJob(ctx context.Context, job *Job) (shoul
 	jobLogger := log.With(c.logger, "groupKey", job.Key())
 	subDir := filepath.Join(c.compactDir, job.Key())
 
+	blockCount := len(job.metasByMinTime)
+
 	defer func() {
 		elapsed := time.Since(jobBeginTime)
 
+		jobType := "merge"
+		if job.UseSplitting() {
+			jobType = "split"
+		}
+
 		if rerr == nil {
-			level.Info(jobLogger).Log("msg", "compaction job succeeded", "duration", elapsed, "duration_ms", elapsed.Milliseconds())
+			c.metrics.compactionJobDuration.WithLabelValues(jobType).Observe(elapsed.Seconds())
+			c.metrics.compactionJobBlocks.WithLabelValues(jobType).Observe(float64(blockCount))
+
+			level.Info(jobLogger).Log("msg", "compaction job succeeded", "duration", elapsed, "duration_ms", elapsed.Milliseconds(), "block_count", blockCount)
```

Comment:
nit: Is it useful to include `jobType` in the log messages since we already have it?

Reply:
good idea, let's do that. It can make loki queries simpler.